sn_transfers/src/wallet/local_store.rs

```diff
@@ -195,7 +153,7 @@ impl LocalWallet {
     }
 
     pub fn balance(&self) -> NanoTokens {
-        self.wallet.balance()
+        self.watchonly_wallet.balance()
     }
```

Comment:
Any chance we can short circuit that one?
We have 4 functions named `balance` they call each other in a deep nested way that makes the code hard to navigate, any chance we can get rid of that `LocalWallet` middle man and directly call you new wallet?

Reply:
You are missing that the `LocalWallet` is a different type of wallet, it's like a hot-wallet (eventually we should rename it) which reuses the impl/features of a watch-only wallet. Nevertheless, a refactoring of the LocalWallet is out of the scope here.